pyaedt/edb.py

```diff
@@ -741,6 +730,63 @@ def edb_value(self, val):
         """
         return self.edb.Utility.Value(val)
 
+    @aedt_exception_handler
+    def _is_file_existing_and_released(self, filename):
+        if os.path.exists(filename):
+            try:
+                os.rename(filename, filename + '_')
+                os.rename(filename + '_', filename)
+                # print 'Access on file "' + filename + '" is available!'
+                return True
+            except OSError as e:
+                # print 'Access-error on file "' + filename + '"! \n' + str(e)
+                return False
+        else:
+            # print 'File "' + filename + '" is not existing!'
+            return False
+
+    @aedt_exception_handler
+    def _is_file_existing(self, filename):
+        if os.path.exists(filename):
+            return True
+        else:
+            return False
+
+    @aedt_exception_handler
+    def _wait_for_file_release(self, timeout=30, file_to_release=None):
+        if not file_to_release:
+            file_to_release = os.path.join(self.edbpath)
+        tstart = time.time()
+        while True:
+            if self._is_file_existing_and_released(file_to_release):
+                # print 'File is released'
```

Comment:
The commented `print`can either be deleted or replaced by writting down the message into the log.